irmaTubePremium/messages.en.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const MESSAGES = {
+    'lang':               'en',
+    'cancel-issuance-message':     'IrmaTUBE registration cancelled.',
```

Comment:
Here and elsewhere: the capitalization is reversed, i.e. not `IrmaTUBE` but `IRMATube`.

Reply:
-.-